static/js/reducers/comments.js

```diff
@@ -51,5 +65,13 @@ export const commentsEndpoint = {
       update.set(key, key === postId ? appendCommentToTree(tree, comment, commentId) : tree)
     })
     return update
+  },
+  patchFunc:           (commentId: string, payload: Object) => api.updateComment(commentId, payload),
+  patchSuccessHandler: (response: Comment, data: Map<string, Array<Comment>>) => {
+    let update = new Map()
+    data.forEach((tree, key) => {
+      update.set(key, key === response.post_id ? updateCommentTree(tree, response) : tree)
+    })
```

Comment:
Missed this comment. I'll change the code for the `new Map`. Do you think the `reducers/comments_test.js` test covers the cases you were referring to?